CODEOWNERS

```diff
@@ -80,6 +80,14 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/listener/http_inspector @yxue @PiotrSikora @lizan
 # attribute context
 /*/extensions/filters/common/expr @kyessenov @yangminzhu @lizan
+# webassembly access logger extensions
+/*/extensions/access_loggers/wasm @jplevyak @PiotrSikora @lizan
```

Comment:
@PiotrSikora John is no longer working on this?

Reply:
No, I'll update `CODEOWNERS`.